README.md

```diff
@@ -3,8 +3,15 @@
 ## Prerequisites
 - `node 12.*`
 - `go 1.17.*`, `unzip`, `curl` (for proto generation)
+- [Keplr Chrome extension](https://www.keplr.app/)
+- `tendermint`
 
 ## Usage
+
+Ensure you have a Tendermint node running locally and that CORS is allowed from any origin in `config.toml`.
```

Comment:
In latest cosmos-sdk cors directives in config.toml is ignored (we are communicating with cosmos-sdk development team on it), so workaround for development for now is some chrome extension to bypass this, like this one https://chrome.google.com/webstore/detail/digfbfaphojjndkpccljibejjbppifbc (you can point it here in README)